script/github-release

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env ruby
```

Comment:
It would be nice to use `hub release create` in the future :wink: - I'm actually using it to upload all our releases.